src/Magento/FunctionalTestingFramework/Console/GenerateTestsCommand.php

```diff
@@ -128,18 +117,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
      * @param boolean $force
      * @param string  $debug
      * @param boolean $verbose
+     * @param boolean $allowSkipped
      * @return array
      * @throws \Magento\FunctionalTestingFramework\Exceptions\TestReferenceException
      * @throws \Magento\FunctionalTestingFramework\Exceptions\XmlException
      */
-    private function createTestConfiguration($json, array $tests, bool $force, $debug, bool $verbose)
-    {
+    private function createTestConfiguration(
+        $json,
+        array $tests,
+        bool $force,
+        $debug,
+        bool $verbose,
+        bool $allowSkipped
+    ) {
```

Comment:
Can we either use "type param" or "param"? e.g. $json and $debug don't have type but the other params use type.

Reply:
I was pattern matching to use `type param`, I can change the ones that don't have it to do the same.

I think this is a static check that we don't have enabled in our framework, @okolesnyk prefers type hinting at the signature so I'm going down that route.